docs/ops/activation/LogSoftmax_5.md

```diff
@@ -0,0 +1,56 @@
+## LogSoftMax <a name="LogSoftmax"></a> {#openvino_docs_ops_activation_LogSoftmax_5}
+
+**Versioned name**: *LogSoftmax-5*
+
+**Category**: *Activation*
+
+**Short description**: LogSoftmax computes the natural logarithm of softmax values for the given input.
+
+**Attributes**
+
+* *axis*
+
+  * **Description**: *axis* represents the axis of which the *LogSoftmax* is calculated. Negative value means counting dimensions from the back.
+  * **Range of values**: positive and negative integer value
+  * **Type**: int
+  * **Default value**: -1
```

Comment:
Apologize, your first version with `1` was correct and `1` is due to our layout NCHW. In TF they have NHWC so they use `-1`